erts/doc/src/erlang.xml

```diff
@@ -2500,6 +2500,16 @@ uncompiled code with the same arity are mapped to the same list by
       </desc>
     </func>
 
+    <func>
+      <name name="get_cookie" arity="1" since="OTP @OTP-17538@"/>
+      <fsummary>Get the magic cookie for a node.</fsummary>
+      <desc>
+        <p>Returns the magic cookie for node <c><anno>Node</anno></c>
+          if the local node is alive,
+          otherwise the atom <c>nocookie</c>.</p>
+      </desc>
```

Comment:
Is get_cookie(LocalNode) same as get_cookie()?

Reply:
Yes.  `get_cookie()` is actually implemented with `get_cookie(node())`.